app/api/api_v1/routers/pesquisa/projeto.py

```diff
@@ -20,8 +20,7 @@
 async def projeto_by_name(
     request: Request,
     projeto_name: str,
-    db=Depends(get_db),
-    current_pessoa=Depends(get_current_active_pessoa),
```

Comment:
Se essa variável não for mais utilizada, remover o import da linha 13: `from app.core.auth import get_current_active_pessoa`